cmd/policy/images.go

```diff
@@ -6,11 +6,15 @@ type ImagesCmd struct {
 	Remote    bool   `name:"remote" short:"r" help:"List policies from a remote registry."`
 	Server    string `name:"server" short:"s" help:"Registry server to connect to." default:"{{ .DefaultDomain }}"`
 	ShowEmpty bool   `name:"show-empty" short:"e" help:"Show policies with no images."`
+	Org       string `name:"organization" short:"o" help:"Show images for an organization."`
```

Comment:
the `required` tag might work here:  https://github.com/alecthomas/kong#supported-tags